CRONAPP-3307 - Caixa de Seleção Multipla não atualiza Fonte

alissonsilva-cronapp opened the pull request:
https://cronapp.atlassian.net/browse/CRONAPP-3307

## Problema:
Após inserir um registro e solicitar atualização da fonte de dados pelo servidor, o item adicionado não é exibido no componente da caixa de seleção multipla.
## Solução:
Inserido em options a opção de checar os dados da fonte de dados que popula o componente da caixa de seleção multipla.

alissonsilva-cronapp committed:
CRONAPP-3307 - Caixa de Seleção Multipla não atualiza Fonte

https://cronapp.atlassian.net/browse/CRONAPP-3307

## Problema:
Após inserir um registro e solicitar atualização da fonte de dados pelo servidor, o item adicionado não é exibido no componente da caixa de seleção multipla.
## Solução:
Inserido em options a opção de checar os dados da fonte de dados que popula o componente da caixa de seleção multipla.

drigores reviewed (changes requested):
Existe risco de quebrar caso não tenha o datasource? deveria ter validação?

alissonsilva-cronapp committed:
CRONAPP-3307 - Caixa de Seleção Multipla não atualiza Fonte

https://cronapp.atlassian.net/browse/CRONAPP-3307

## Problema:
Após inserir um registro e solicitar atualização da fonte de dados pelo servidor, o item adicionado não é exibido no componente da caixa de seleção multipla.
## Solução:
Inserido em options a opção de checar os dados da fonte de dados que popula o componente da caixa de seleção multipla.

alissonsilva-cronapp commented:
Adicionado validação

drigores reviewed (changes requested):
Need validation on options.dataSource.transport.options.cronappDatasource.data.

alissonsilva-cronapp committed:
CRONAPP-3307 - Caixa de Seleção Multipla não atualiza Fonte

https://cronapp.atlassian.net/browse/CRONAPP-3307

## Problema:
Após inserir um registro e solicitar atualização da fonte de dados pelo servidor, o item adicionado não é exibido no componente da caixa de seleção multipla.
## Solução:
Inserido em options a opção de checar os dados da fonte de dados que popula o componente da caixa de seleção multipla.

tiagoromano commented on js/directives.js:
```diff
@@ -4003,12 +4003,22 @@
 
             var combobox = $element.kendoMultiSelect(options).data('kendoMultiSelect');
+
+            combobox.bind("open", (_e) => {
```
Alisson, eu acredito que o melhor lugar para resolver não seja ai, você verificou no datasource gerado para o kendo (função -> app.kendoHelper.getDatasource), la tem uma parte do push, que notifica o componente que veio novo dado e chama automaticamente o read, acredito que o problema esteja lá, verifica isso por favor.

tiagoromano reviewed (commented):
Da uma olhada no comentário
